docs/api.md

```diff
@@ -508,6 +508,18 @@ when using the `transport` option. In this case an `Error` will be thrown.
 
 * See [pino.transport()](#pino-transport)
 
+#### `onChild` (Function)
+
+The `onChild` function is a callback that will be called on each creation of a new child and use the child instance as it's first argument.
```

Comment:
Does the callback block child creation? Do we handle any errors that occur within the callback?

Reply:
Currently no. Should we wrap the callback in a `try-catch` block?

Reply:
I think the way the callback is handled should be made clear in the documentation. Adding a `try/catch` during child creation will have a performance penalty. So we would want to alter the documentation to make it clear that any error in the callback will be uncaught.

Reply:
Got it. I've updated the documentation to mention the callback doesn't handle errors.